rustls/src/quic.rs

```diff
@@ -705,6 +705,28 @@ pub trait PacketKey: Send + Sync {
 
     /// Tag length for the underlying AEAD algorithm
     fn tag_len(&self) -> usize;
+
+    /// Number of QUIC messages that can be safely encrypted with a single key of this type.
+    ///
+    /// Once a `MessageEncrypter` produced for this suite has encrypted more than
+    /// `confidentiality_limit` messages, an attacker gains an advantage in distinguishing it
+    /// from an ideal pseudorandom permutation (PRP).
+    ///
+    /// This is to be set on the assumption that messages are maximally sized --
+    /// 2 ** 16. For non-QUIC TCP connections see [`CipherSuiteCommon::confidentiality_limit`][csc-limit].
+    ///
+    /// [csc-limit]: crate::crypto::CipherSuiteCommon::confidentiality_limit
+    fn confidentiality_limit(&self) -> u64;
+
+    /// Number of QUIC messages that can be safely decrypted with a single key of this type
+    ///
+    /// Once a `MessageDecrypter` produced for this suite has failed to decrypt `integrity_limit`
+    /// messages, an attacker gains an advantage in forging messages.
+    ///
+    /// This is not relevant for TLS over TCP (which is implemented in this crate)
+    /// because a single failed decryption is fatal to the connection.  However,
+    /// this quantity is used by QUIC.
```

Comment:
This is a bit confusing - looks like a copy-paste from the older comment.
In particular - the `which is implemented in this crate` part. Would probably be better as `which is also implemented in this crate`.

Reply:
> Would probably be better as which is also implemented in this crate (added also).

:+1: Want to open a PR with the change?